applications/inventory-app/src/plugins/store.js

```diff
@@ -0,0 +1,376 @@
+/*
+    store.js
+
+    Created by Kalila L. on 16 Apr 2020.
+    Copyright 2020 Vircadia and contributors.
+    
+    Distributed under the Apache License, Version 2.0.
+    See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+*/
+
+import Vue from 'vue';
+import Vuex from 'vuex';
+
+Vue.use(Vuex);
+
+export const store = new Vuex.Store({
+    devtools: true,
+    state: {
+        items: [
+            {
+                "type": "script",
+                "name": "VRGrabScale",
+                "url": "https://gooawefaweawfgle.com/vr.js",
```

Comment:
Presumably all this is test data?
If it can't be deleted, add a comment explaining how it's used for testing, and how it's not included in the "real" app in productcion.